.eslintrc.json

```diff
@@ -18,7 +18,7 @@
     "plugins": ["react", "@typescript-eslint"],
     "rules": {
         "linebreak-style": ["error", "unix"],
-        "quotes": ["error", "single"],
+        "quotes": ["error", "single", { "avoidEscape": true }],
```

Comment:
Why are we adding this rule here?
We should follow one

Reply:
![image](https://github.com/Real-Dev-Squad/website-status/assets/107541780/78d28778-0568-48d5-982a-254ebea781c5)
if i doesn't add it to eslint, eslint is trying to escape character & prettier is trying to avoid escaping characters, resulting in git push failure when i try to push changes

Reply:
In that case, we should fix the prettier and eslint configurations

Reply:
> In that case, we should fix the prettier and eslint configurations, rather than just adding some rule to make precommits pass

i searched a lot to fix that issue but not found any solution for it, so i reverted my code to use defalut lint rules

Reply:
if you face a issue that is not linked with your PR, please create a separate issue ticket and PR for it